app/ui-react/packages/ui/src/Integration/IntegrationDetailHistoryListViewItemActions.tsx

```diff
@@ -0,0 +1,54 @@
+import * as H from 'history';
+import { DropdownKebab } from 'patternfly-react';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+export interface IIntegrationMenuActions {
```

Comment:
This is the other interface I'm talking about, it's not specific in either case so I don't think it warrants a duplicate interface.

Reply:
You got it, makes sense. 👍